packit/constants.py

```diff
@@ -206,14 +206,17 @@
     "git clone ssh://$YOUR_USER@pkgs.fedoraproject.org/forks/{user}/rpms/{package}.git\n"
     "cd {package}\n"
     "git checkout {branch}\n"
+    "git push origin {branch}\n"
     "```\n"
     "\n---\n\n"
-    "Alternatively, you can locally fetch the source branch of this PR "
-    "from Packit's fork and push it (with a fix) to your fork:\n"
```

Comment:
let me know if you think we should still have this in the instructions

Reply:
I would probably remove this since the description is getting really long.